sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -235,6 +235,54 @@ def compute_table_name(row):
 [2] https://cloud.google.com/bigquery/docs/reference/rest/v2/tables/insert
 [3] https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#resource
 
+Output of the WriteToBigQuery transform
+---------------------------------------
+
+Writing to BigQuery returns a WriteResult object that includes metadata
+relating to the write you configured. This data can be used in later steps
+in your pipeline:::
+
+  schema = {'fields': [
+      {'name': 'column', 'type': 'STRING', 'mode': 'NULLABLE'}]}
+
+  error_schema = {'fields': [
+      {'name': 'destination', 'type': 'STRING', 'mode': 'NULLABLE'},
+      {'name': 'row', 'type': 'STRING', 'mode': 'NULLABLE'},
+      {'name': 'error_message', 'type': 'STRING', 'mode': 'NULLABLE'}]}
+
+  with Pipeline() as p:
+    result = (p
+      | 'Create Columns' >> beam.Create([
+              {'column': 'value'},
+              {'bad_column': 'bad_value'}
+            ])
+      | 'Write Data' >> WriteToBigQuery(
+              method=WriteToBigQuery.Method.STREAMING_INSERTS,
+              table=my_table,
+              schema=schema,
+              insert_retry_strategy=RetryStrategy.RETRY_NEVER
+            ))
+
+    _ = (result.failed_rows_with_errors
+      | 'Get Errors' >> beam.Map(lambda e: {
+              "destination": e[0],
+              "row": json.dumps(e[1]),
+              "error_message": e[2][0]['message']
+            })
+      | 'Write Errors' >> WriteToBigQuery(
+              method=WriteToBigQuery.Method.STREAMING_INSERTS,
+              table=error_log_table,
+              schema=error_schema,
+            ))
+
+Attributes can be accessed using dot notation or bracket notation:
+
+result.failed_rows                  <--> result['FailedRows']
+result.failed_rows_with_errors      <--> result['FailedRowsWithErrors']
+result.destination_load_jobid_pairs <--> result['destination_load_jobid_pairs']
+result.destination_file_pairs       <--> result['destination_file_pairs']
+result.destination_copy_jobid_pairs <--> result['destination_copy_jobid_pairs']
```

Comment:
This looks great. Thanks! Do you know whether lines 280-284 show up as code/monospace font? Maybe you need to add backticks or indentantion to make it look as code?

Finally, let's add a sub-section that says something like this:

```
### Chaining of operations after WriteToBigQuery

WriteToBigQuery returns an object with several PCollections with metadata about the write operations. These are
usfeul to inspect the write operation; however, often the simplest use case is to chain an operation after writing data
to BigQuery. 

To chain an operation, one can chain it after one of the output PCollections. A generic way in which one could chain
this operation (independent of write method) would be:::

  def chain_after(result):
    try:
      # This should work for FILE_LOADS where we run load and possibly copy jobs.
      return (result.load_jobid_pairs, result.copy_jobid_pairs) | beam.Flatten()
    except AttributeError:
      return result.failed_rows

  result = (pcoll | WriteToBigQuery(...))

  _ = (chain_after(result) 
       | beam.Reshuffle()  # Force a 'commit' of the intermediate date
       | MyOperationAfterWriteToBQ())
```